test/requirements-dev.txt

```diff
@@ -0,0 +1 @@
+treon==0.1.2
```

Comment:
Should add: `-r requirements.txt` to this line in order to include those dependencies as well.  Example:
https://github.com/HumanCellAtlas/data-store/blob/master/requirements-dev.txt.in#L14